manifests/nfd-topology-updater-job.yaml.template

```diff
@@ -0,0 +1,77 @@
+apiVersion: batch/v1
```

Comment:
there are a few differences on how the pod is called in the `Job` vs `DaemonSet`, the more important, the Volume Mounts, leading to failures on the `Job` due not mouting podresources

also one defines a namespace to watch `rte` and the other goes to `all namespaces`

I think we should standarize the way we call the pod on the yaml templates

Reply:
Ack, will make sure that the job and daemonset is consistent.